Source/WTF/WTF.xcodeproj/project.pbxproj

```diff
@@ -4073,7 +4073,7 @@
 				1CA85CA9241B0B260071C2F5 /* RuntimeApplicationChecksCocoa.cpp in Sources */,
 				46E93049271F1205005BA6E5 /* SafeStrerror.cpp in Sources */,
 				A3EE5C3D21FFAC7D00FABD61 /* SchedulePairCF.cpp in Sources */,
```

Comment:
I'm curious, why is there a different file for Cocoa and for CF ?

Reply:
`SchedulePairCF.cpp` contains an equality operator that understand CFString. It has no other content.